FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -2261,6 +2261,17 @@ - (nullable FIRUser *)getStoredUserForAccessGroup:(NSString *_Nullable)accessGro
 #endif  // TARGET_OS_WATCH
     user = [unarchiver decodeObjectOfClass:[FIRUser class] forKey:userKey];
   } else {
+#if TARGET_OS_TV
+    if (self.shareAuthStateAcrossDevices) {
+      FIRLogError(
+          kFIRLoggerAuth, @"I-AUT000001",
+          @"Due to tvOS system limitations, getting a stored user for a given access "
+          @"group is not supported on tvOS when sharing auth state across devices "
+          @"(`shareAuthStateAcrossDevices`) is enabled (#8878). This case will return `nil`. ");
```

Comment:
run-on sentence? Perhaps a period at the end of 2269?  same below.

Reply:
Thanks. I was trying to be precise but it's more verbose than anything. I reworked here and below in 2a944f7.

Reply:
Hmm. I still think there needs to be punctuation after "across devices"

Reply:
Ah sorry, I simplified further in f7977ca